check-ecsync-optimism.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/sh
+SYNC=$(curl -s -m2 -N -X POST -H "Content-Type: application/json" --data '{"jsonrpc":"2.0","method":"eth_syncing","params":[],"id":1}' "https://${HAPROXY_SERVER_NAME}")
```

Comment:
`eth_syncing` isn't currently supported with l2geth atm.
It should always return `false`, have you observed something different?

Reply:
Good point. I am pointing this out in the README now. Can you think of a better endpoint to verify l2geth is synced?